nerfstudio/viewer_legacy/viser/server.py

```diff
@@ -44,7 +44,7 @@ def __init__(
     ):
         super().__init__()
 
-        self._ws_server = viser.infra.Server(host, port, http_server_root=None, verbose=False)
+        self._ws_server = ViserServer(host, port, http_server_root=None, verbose=False)
```

Comment:
sorry, I think this is now renamed to: https://viser.studio/latest/infrastructure/#viser.infra.WebsockServer

Reply:
change is made